### ISSUE TITLE
Replace Method is applied for connection that doesn't support it.

### ISSUE THREAD

**Issue by uslss** [dbeaver/dbeaver#12621]
#### System information: 
- Operating system (distribution) and version win10
- DBeaver version 21.1.0

#### Connection specification:
- Database name and version SQL Server, SQLite

#### Describe the problem you're observing:
Replace Method is applied for connection that doesn't support it.
#### Steps to reproduce, if exist:
1. Start import to the table from CSV (SQLite)
2. Choose Ignore or replace Replace method and start import - import is successfully performed
3. Start import to the table from CSV (SQL Server)
4. Replace method is greyed out and it cannot be chosen.
5. Start import - an error appears:

```
An error occurred during data load

Reason:
SQL Error [156] [S0001]: Incorrect syntax near the keyword 'OR'.
```

Note: Error doesn't appear if None replace method was chosen during the import to SQLite
![2021-05-27_14h07_38](https://user-images.githubusercontent.com/31996417/119816970-0d577600-bef6-11eb-8b03-c8985ecdc75c.gif)

**Comment by LonwoLonwo** [comment]
Fixed in DBeaver 21.1.0

**Comment by uslss** [comment]
verified
